DigitalOffice/Tests/AuthenticationServiceTests/TokenTests/SigningEncodingKeyTests.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Text;
+using NUnit.Framework;
+using Microsoft.IdentityModel.Tokens;
+using LT.DigitalOffice.AuthenticationService.Token;
+using LT.DigitalOffice.AuthenticationService.Token.Interfaces;
+
+namespace LT.DigitalOffice.AuthenticationServiceUnitTests.TokenTests
```

Comment:
It is necessary to change the name from TokenTests to Token.

Reply:
Fixed